~ Sept 28: New 'domain capture' doc (#9407)
newrelic/docs-website: src/content/docs/accounts/accounts-billing/account-setup/domain-capture.mdx

---
title: "Domain capture: Capture your users when they sign up for New Relic"
tags:
  - Accounts
  - Accounts and billing
  - Account setup
metaDescription: "When people in your organization attempt to sign up for New Relic, you can use domain capture to make sure they're routed into your account."
redirects:
---

import newUserLogin from 'images/domain-capture-new-user-login.png'

import settings from 'images/domain-capture-settings.png'

import dnsVerification from 'images/domain-capture-dns-verification.png'

> When you have a [New Relic organization](/docs/accounts/accounts-billing/account-structure/new-relic-account-structure), sometimes your users won't know they should be using that organization and will accidentally sign up for New Relic and create a new, separate New Relic organization. This can result in confusion and wasted time. 

Reviewer: ```suggestion
When part of a [New Relic organization](/docs/accounts/accounts-billing/account-structure/new-relic-account-structure), you may accidentally sign up for New Relic and create a new, separate New Relic organization. This results in confusion and wasted time. 
```
The original sentence was a bit runon-y, so I've changed who the 'you' is from an owner of an org to the user who accidentally creates a new org. Whatever you end up deciding, simplifying this sentence will help identify the problem the doc is solving quicker.

> With **domain capture**, you can register your domains with New Relic. When you do that, users in your organization who attempt to sign up for New Relic will be given the option to join your New Relic organization. You can also control how those users are routed into New Relic: their user type and user group. The larger your organization is, the more useful domain capture will be. 

Reviewer: Can you include a short technical sentence about what domain capture is? You've covered what the end result is (clearly delineated areas for users to choose the right option), which is the most important part. I'm just curious about where this is happening and what the high level overview of the process looks like.
